lib/teleterm/api/proto/v1/usage_events.proto

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.terminal.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
+
+message ReportEventRequest {
+  string distinct_id = 1;
+  google.protobuf.Timestamp timestamp = 2;
+  ConnectUsageEventOneOf event = 3;
+}
+
+message EventReportedResponse {}
```

Comment:
This can also be an `EmptyResponse`

Reply:
Done, I think it's ok to use our common `EmptyResponse` instead of `google.protobuf.Empty`.